.idea/runConfigurations/worldwind_androidTest.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <component name="ProjectRunConfigurationManager">
```

Comment:
Change needed: Any idea why this Android Studio config file has changed?

Reply:
I don't know why this changed. I will revert it back.